download-maven-plugin/src/main/java/com/googlecode/WGet.java

```diff
@@ -145,26 +145,32 @@
 
   /**
    * Whether to skip execution of Mojo
-   * 
+   *
    * @parameter expression="${download.plugin.skip}" default-value="false"
    */
   private boolean skip;
 
   /**
+   * Whether to check the signature of existing files
+   * @parameter expression="${checkSignature}" default-value="false"
+   */
+  private boolean checkSignature;
```

Comment:
Actually, I'm wondering whether this should even be a parameter. Don't you think the check you suggest should be always performed?